frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/LookAndFeelSettings.tsx

```diff
@@ -52,7 +52,10 @@ export const LookAndFeelSettings = ({
     }),
   );
   const upgradePageUrl = useSelector(state =>
-    getUpgradeUrl(state, { utm_media: "static-embed-settings-appearance" }),
+    getUpgradeUrl(state, {
```

Comment:
This is a change for
Interactive embedding upsell
Overview doc
Look and Feel doc
✅ Look and Feel font upsell